integration-cli/docker_cli_rm_test.go

```diff
@@ -40,6 +40,8 @@ func (s *DockerSuite) TestRmForceRemoveRunningContainer(c *check.C) {
 	dockerCmd(c, "rm", "-f", "foo")
 }
 
+// FIXME: I have no idea what the point of this test case is. What does it mean
+// for a container to be "orphaned"?
```

Comment:
Okay, so it's an "image orphaning" test rather than a "container orphaning" test. The name makes it sound like it has something to do with process orphaning.